sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -0,0 +1,197 @@
+# Azure Cognitive Services Form Recognizer client library for .NET
+Azure Cognitive Services Form Recognizer is a cloud service that uses machine learning to extract text and table data from form documents. It allows you to train custom models using your own forms, to extract field names and values, and table data from them.  It also provides a prebuilt models you can use to extract values from receipts, or tables from any form.
+
+[Source code][formreco_client_src] | <!--[Package (NuGet)]() | [API reference documentation]() |--> [Product documentation][formreco_docs] <!--| [Samples]()-->
+
+## Getting started
+
+### Prerequisites
+* An [Azure subscription][azure_sub].
+* An existing Form Recognizer resource. If you need to create the resource, you can use the [Azure Portal][azure_portal] or [Azure CLI][azure_cli].
+<!-- 
+If you use the Azure CLI, replace `<your-resource-group-name>`, `<your-resource-name>`, `<location>`, and `<sku>` with your values:
+
+```PowerShell
+az cognitiveservices account create --kind TextAnalytics --resource-group <your-resource-group-name> --name <your-resource-name> --location <location> --sku <sku>
+``` -->
+<!-- 
+### Install the package
+Install the Azure Text Analytics client library for .NET with [NuGet][nuget].  To use the [.NET CLI](https://docs.microsoft.com/en-us/nuget/consume-packages/install-use-packages-dotnet-cli), run the following command from PowerShell in the directory that contains your project file:
+
+```PowerShell
+dotnet add package Azure.AI.TextAnalytics --version 1.0.0-preview.1
+``` 
+For other installation methods, please see the package information on [NuGet][nuget].
+-->
+
+### Authenticate a Form Recognizer client
+In order to interact with the Form Recognizer service, you'll need to select either a `ReceiptClient`, `FormLayoutClient`, or `CustomFormClient`, and create an instance of this class.  In the following samples, we will use CustomFormClient as an example.  You will need an **endpoint**, and either a **subscription key** or ``TokenCredential`` to instantiate a client object.  For more information regarding authenticating with cognitive services, see [Authenticate requests to Azure Cognitive Services][cognitive_auth].
```

Comment:
I realize this is mostly a WIP, but we might want to say API key instead of subscription key like we do in text analytics